matcher/matching.go

```diff
@@ -1,3 +1,26 @@
 // Package matcher provides matching positioning messages to a set of filtering rules
 // and pereating messages to a topic in case of they are matched.
 package matcher
+
+import (
+	"github.com/lvl484/positioning-filter/position"
+	"github.com/lvl484/positioning-filter/repository"
+)
+
+type Matcher interface {
+	Match(position.Position) (bool, error)
+}
+
+type matcher struct {
+	filters repository.Filters
```

Comment:
It will be better to provide position and filter (config from repository) to matcher as parameters

Reply:
Well, i have no time to rewrite logic of components